src/dune_engine/sandbox.ml

```diff
@@ -2,6 +2,11 @@ open Import
 
 let sandbox_dir = Path.Build.relative Path.Build.root ".sandbox"
 
+let async_sandboxing f =
+  match Config.(get background_sandboxes) with
```

Comment:
You could probably move the `match` to the top level.

Reply:
I wish. But then it would mean that the feature wouldn't be configurable through dune-config. If we could live with configurations being set through the environment, we could do it at the toplevel though.